src/Silverback.Integration.Kafka/Messaging/Broker/Kafka/ConfluentConsumerBuilder.cs

```diff
@@ -13,6 +13,15 @@ namespace Silverback.Messaging.Broker.Kafka
     /// </summary>
     public class ConfluentConsumerBuilder : IConfluentConsumerBuilder
     {
+        /// <summary>
+        ///     If the KafkaConsumer does not have a group.id for consumer group partition assignment and offset
+        ///     storage, then a fake value is set as workaround for the current limitations of confluent-kafka-dotnet and librdkafka.
+        ///     See also
+        ///     <see href="https://github.com/confluentinc/confluent-kafka-dotnet/issues/225" />
+        ///     <see href="https://github.com/edenhill/librdkafka/issues/593" /> for more information.
+        /// </summary>
+        public const string UnusedGroupId = "unused";
```

Comment:
Does it need to be public?

I don't like the name _"unused"_. This is the dummy group id you use when not set by the user and _"unused"_ doesn't tell me this story.

Reply:
Is it a valid alternative?
https://github.com/BEagle1984/silverback/blob/fa38eb8ef7de7181003ac5efdf2b6bba42be1cb9/src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/ConfluentClientConfigProxies.cs#L886

Reply:
I like it more, yes. The actual string could also be just "not-set", but I don't mind it either way.